### DISCUSSION TITLE
Show full commit number when view commit history of a branch (particularly master/main branch)

### DISCUSSION THREAD

**Discussion by sh977218** [community/community#106842] [Repositories] [answered]
### Select Topic Area

Show & Tell

### Body

In a recently debug tracing process, I have to find out which commit cause a bug. And unfortunately that commit is way back, and I have `git checkout` each PR's merge commit and find out at which commit number is the root cause.
When I view the branch's commit history, it look like this screenshot, the copy commit is very easy and convenient, but the commit number is partial hidden. Maybe we can have a toggle somewhere on this page to show all full commit number. 
![image](https://github.com/community/community/assets/5692380/3db14782-6db9-42da-b004-8822d9aae0b7)

**Comment by gil-son** [comment]
Hello,

Open the terminal in the directory of your project and use that command:

```
git log
```

**Comment by MattiaValerio** [comment, marked as answer]
Try with this:
`git log <branch-name>`
It should do the work, let me know
